Headers/AppKit/NSView.h

```diff
@@ -672,10 +679,61 @@ PACKAGE_SCOPE
 - (BOOL) translatesAutoresizingMaskIntoConstraints;
 - (void) setTranslatesAutoresizingMaskIntoConstraints: (BOOL)translatesAutoresizingMaskIntoConstraints;
 #endif
+
+#if GS_HAS_DECLARED_PROPERTIES
+@property float contentCompressionResistancePriority;
+#else
+- (BOOL) contentCompressionResistancePriority;
+- (void) setContentCompressionResistancePriority: (float)priority;
 #endif
 
+#if GS_HAS_DECLARED_PROPERTIES
+@property (readonly) NSSize intrinsicContentSize;
+#else
+- (BOOL) intrinsicContentSize;
+#endif
+
+#if GS_HAS_DECLARED_PROPERTIES
+@property (readonly) CGFloat baselineOffsetFromBottom;
+#else
+- (CGFloat) baselineOffsetFromBottom;
+#endif
+
+#if GS_HAS_DECLARED_PROPERTIES
+@property (readonly) CGFloat firstBaselineOffsetFromTop;
+#else
+- (CGFloat) firstBaselineOffsetFromTop;
+#endif
+
+#endif
+
+@end
+
+#if OS_API_VERSION(MAC_OS_X_VERSION_10_7, GS_API_LATEST)
+@interface NSView (NSConstraintBasedLayoutCoreMethods)
+
+- (void) updateConstraints;
+
+- (void) updateConstraintsForSubtreeIfNeeded;
+
+- (NSLayoutPriority)contentHuggingPriorityForOrientation:(NSLayoutConstraintOrientation)orientation;
```

Comment:
You could use a few more spaces here to be consistent with your other methods. I know that the GNustep coding guidelines aren't that consistent by themselves. At least within a block we could try :-)

Reply:
Fixed

Reply:
Somehow I cannot see any changes here. Could you please recheck?